src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceRentDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
```

Comment:
Just a heads-up that it's a good practice in JavaScript to use SNAKE_CASE (all uppercase) for constant variables.
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/46566190/17912edd-4fa4-41ed-858e-8ed98ec12c10)